scripts/bet_omen.py

```diff
@@ -26,14 +38,48 @@ def main(
     """
     market = get_market(market_id)
     omen_buy_outcome_tx(
-        amount=xDai(amount),
+        amount=xdai_type(amount),
+        from_address=HexAddress(from_address),
+        from_private_key=PrivateKey(from_private_key),
+        market=market,
+        outcome=outcome,
+        auto_deposit=auto_deposit,
+    )
+
+
+@app.command()
+def sell(
+    amount: str = typer.Option(),
+    from_address: str = typer.Option(),
+    from_private_key: str = typer.Option(),
+    market_id: str = typer.Option(),
+    outcome: str = typer.Option(),
+    auto_withdraw: bool = typer.Option(False),
+) -> None:
+    """
+    Helper script to sell outcome of an existing bet on Omen, usage:
+
+    ```bash
+    python scripts/bet_omen.py sell \
+        --amount 0.01 \
+        --from-address your-address \
+        --from-private-key your-private-key \
+        --market-id some-market-id \
+        --outcome one-of-the-outcomes
+    ```
+
+    Market ID can be found easily in the URL: https://aiomen.eth.limo/#/0x86376012a5185f484ec33429cadfa00a8052d9d4
+    """
+    market = get_market(market_id)
+    omen_sell_outcome_tx(
+        amount=xdai_type(amount),
         from_address=HexAddress(from_address),
         from_private_key=PrivateKey(from_private_key),
         market=market,
         outcome=outcome,
-        auto_deposit=True,
+        auto_withdraw=auto_withdraw,
     )
 
 
 if __name__ == "__main__":
-    typer.run(main)
```

Comment:
In case you didn't work with Typer before, it does pretty cool stuff out of the box: 
![Screenshot by Dropbox Capture](https://github.com/evangriffiths/prediction-market-agent/assets/15619339/ee58597c-3e9a-448a-8df8-b7e5f52a6454) 
![Screenshot by Dropbox Capture](https://github.com/evangriffiths/prediction-market-agent/assets/15619339/f63a14b9-1799-4afd-a5c0-cf4fb680a4d6)

Reply:
Oo I haven't used it before. Very nice!